srcpkgs/brother-brscan5/REMOVE

```diff
@@ -0,0 +1,10 @@
+case "${ACTION}" in
+    pre)
+	# remove brother driver from sane configuration
+	_TEMPFILE=`mktemp sane_dll_conf_XXXXXXXXXXX`
+	_SANE_CONF='/etc/sane.d/dll.conf'
+
+	( cat ${_SANE_CONF} | sed -e '/brother5/d' > ${_TEMPFILE} ) && cat ${_TEMPFILE} > ${_SANE_CONF}
```

Comment:
This could be simplified with `-i`. 

`sed -i -e '/brother5/d' /etc/sane.d/dll.conf`

Reply:
Again I copied the brother-brscan4 package here. I have no particular preference so I'm happy to use `-i` here if you think it's better.

Reply:
It saves having to bother with a tempfile.

Reply:
Ok, done.